src/function.rs

```diff
@@ -460,4 +462,24 @@ impl Function {
   pub fn set_name(&self, name: Local<String>) {
     unsafe { v8__Function__SetName(self, &*name) }
   }
+
+  /// Get the (zero-indexed) column number of the function's definition, if available.
+  pub fn get_script_column_number(&self) -> Option<int> {
+    let ret = unsafe { v8__Function__GetScriptColumnNumber(self) };
+    if ret < 0 {
+      None
+    } else {
+      Some(ret)
+    }
+  }
+
+  /// Get the (zero-indexed) line number of the function's definition, if available.
+  pub fn get_script_line_number(&self) -> Option<int> {
+    let ret = unsafe { v8__Function__GetScriptLineNumber(self) };
+    if ret < 0 {
+      None
+    } else {
+      Some(ret)
+    }
+  }
```

Comment:
```suggestion
  pub fn get_script_column_number(&self) -> Option<u32> {
    let ret = unsafe { v8__Function__GetScriptColumnNumber(self) };
    (ret >= 0).then(|| ret as u32)
  }

  /// Get the (zero-indexed) line number of the function's definition, if available.
  pub fn get_script_line_number(&self) -> Option<u32> {
    let ret = unsafe { v8__Function__GetScriptLineNumber(self) };
    (ret >= 0).then(|| ret as u32)
```
`int` is `std::os::raw::c_int` and shouldn't leak out into the public API when it can be avoided.

I picked `u32` but, depending on what you do with the return value, `usize` might be more appropriate.